inginious/frontend/i18n/es/LC_MESSAGES/messages.po

```diff
@@ -1672,6 +1672,167 @@ msgstr "Enviar"
 msgid "Grader"
 msgstr "Calificador"
 
+#: inginious/frontend/plugins/grader_generator/pages/grader_form.py:25
+msgid "'Maximum diff lines' must be an integer"
+msgstr "'Máximo líneas de diferencia' debe ser entero"
+
+#: inginious/frontend/plugins/grader_generator/pages/grader_form.py:31
+msgid "'Diff context lines' must be an integer"
+msgstr "'Líneas de diferencia de contexto' debe ser un entero"
+
+#: inginious/frontend/plugins/grader_generator/pages/grader_form.py:39
+msgid "Grader: the problem was not specified"
+msgstr "Calificador: el problema no fue especificado"
+
+#: inginious/frontend/plugins/grader_generator/pages/grader_form.py:43
+msgid "Grader: problem does not exist"
+msgstr "Calificador: el problema no existe"
+
+#: inginious/frontend/plugins/grader_generator/pages/grader_form.py:50
+msgid ""
+"Grader: only 'Code Multiple Language', 'Code File Multiple Language' and "
+"'Notebook' problems are supported"
+msgstr ""
+"Calificador: Solo 'Code Multiple Language', 'Code File Multiple Language'"
+" y 'Notebook' son soportados."
+
+#: inginious/frontend/plugins/grader_generator/pages/grader_form.py:54
+msgid "'Maximum diff lines' must be positive"
+msgstr "'Máximo líneas de diferencia' debe ser positivo."
```

Comment:
Maybe  `Líneas de diferencia máxima` is better, but I leave the previous options just in case.